RepoUtilities/pre-commit.py

```diff
@@ -0,0 +1,30 @@
+import json
+import sys
+import os
+import re
+
+def FailCommit(reason):
+    print("Oh no! Bad commit! "+reason)
+    sys.exit(1)
+
+def GetStagedFileContent(filePath):
+    cmd = 'git show :' + filePath
+    stream = os.popen(cmd)
+    return stream.read()
+
+def GetLFSStagedFileContent(filePath):
+    cmd = f"git cat-file blob :{filePath} | git lfs smudge"
+    stream = os.popen(cmd)
+    return stream.read()
+
+output = GetStagedFileContent('Packages/manifest.json')
+packages = json.loads(output)['dependencies']
+if ('com.unity.multiplayer.virtualprojects' in packages):
+    FailCommit("Virtual projects in packages")
+if ('github' in packages['com.unity.multiplayer.tools'].lower()):
+    FailCommit("Tools using github package")
```

Comment:
Is there an easy way to bypass this? Tools has public releases and there will be valid commits periodically to reference new versions

Reply:
OH - I misread this line, I didn't see the `'github'` part. This is fine!